### PR TITLE
add travel-journal

### PR DESCRIPTION
**Pull request by nngrz**
Result
Screenshot 2024-08-19 at 23 29 25

### CODE REVIEW

**Review by balinskia — COMMENTED** [review]
Good job! The code starting too look serious!

File: digital_business_card/public/index.html [hunk 1]
```diff
@@ -3,7 +3,6 @@
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
[the remark this answers is not in the document]
Comment (balinskia) [reply digital_business_card/public/index.html:6]: Revert this change. Your or is about travel journal, not about business card. In general, in PRs, you shouldn't mix two projects. One PR - one project

File: airbnb_experience/package-lock.json
Comment (balinskia) [inline airbnb_experience/package-lock.json:1]: please remove changes from airbnb-experience from this PR

File: travel_journal/src/components/About.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faLocationDot } from "@fortawesome/free-solid-svg-icons"
+
+export default function About(props) {
```
Comment (balinskia) [inline travel_journal/src/components/About.js:5]: Two options:

1. you can destruct it: `About({item})`
and then use: `item.page`

2.  you can destruct it: `About({item: { page, name, startDate...}})`
and then use: `page`

File: airbnb_experience/package-lock.json
Comment (nngrz) [reply airbnb_experience/package-lock.json:1]: i just changed name

Comment (balinskia) [reply airbnb_experience/package-lock.json:1]: Please do it in a different PR

Comment (nngrz) [reply airbnb_experience/package-lock.json:1]: so i have to change the name back?

Comment (balinskia) [reply airbnb_experience/package-lock.json:1]: Yes. please revert changes from the airbnb-experience folder from this PR.
It is very important to send your PRs to review with only the necessary code that is needed and nothing more.

Comment (balinskia) [reply airbnb_experience/package-lock.json:1]: So that once I go to the files-changed tab in the PR I can only see the files related to travel

**Review by balinskia — APPROVED** [review]

**Pull request merged**